packages/expo-video/ios/VideoView.swift

```diff
@@ -108,6 +119,20 @@ public final class VideoView: ExpoView, AVPlayerViewControllerDelegate {
     }
   }
 
+  // MARK: - Lifecycle
+
+  func onAppBackgrounded() {
+    if staysActiveInBackground {
+      setPlayerTracksEnabled(enabled: isInPictureInPicture || false)
+    } else if !isInPictureInPicture {
+      player?.pause()
+    }
+  }
+
+  func onAppForegrounded() {
+    setPlayerTracksEnabled(enabled: true)
+  }
```

Comment:
What do you think if we move this logic into `VideoViewManager`? I think the view itself shouldn't be responsible for what to do when the app goes to background/foreground. Let the manager manage that behavior 😉